code/datums/status_effects/buffs.dm

```diff
@@ -188,15 +188,55 @@
 		return
 	REMOVE_TRAIT(owner, TRAIT_CHUNKYFINGERS, VAMPIRE_TRAIT)
 	var/mob/living/carbon/human/H = owner
-	H.physiology.brute_mod /= 0.5
-	H.physiology.burn_mod /= 0.8
+	H.physiology.brute_mod /= 0.4
+	H.physiology.burn_mod /= 0.5
 	H.physiology.stamina_mod /= 0.5
 	H.physiology.stun_mod /= 0.5
 	if(bonus_damage_applied)
 		bonus_damage_applied = FALSE
 		H.physiology.melee_bonus -= 10
 		H.dna.species.punchstunthreshold -= 8
 
+/datum/status_effect/vampire_gladiator
+	id = "vampire_gladiator"
+	duration = 30 SECONDS
+	tick_interval = 1 SECONDS
+	alert_type = /obj/screen/alert/status_effect/vampire_gladiator
+
+/obj/screen/alert/status_effect/vampire_gladiator
+	name = "Gladiatorial Resilience"
+	desc = "Roused by the thrill of the fight, your body has become more resistant to breaking!"
+	icon = 'icons/mob/actions/actions.dmi'
+	icon_state = "mech_damtype_brute"
+
+/datum/status_effect/vampire_gladiator/on_apply()
+	. = ..()
+	var/mob/living/carbon/human/H = owner
+	if(!istype(H))
+		return
+	ADD_TRAIT(H, TRAIT_NO_BONES, GLADIATOR)
+	ADD_TRAIT(H, TRAIT_STURDY_LIMBS, GLADIATOR)
+	ADD_TRAIT(H, TRAIT_BURN_WOUND_IMMUNE, GLADIATOR)
+	for(var/obj/item/organ/external/limb in H.bodyparts)
+		limb.add_limb_flags()
+		limb.limb_flags |= CANNOT_INT_BLEED
```

Comment:
ok, adding this bitflag manually like this means that it cannot be used anywhere in the future.
lets say we add a chemical that adds this flag to all your limbs, and you take that chemical, then use this vampire power, then the chemical runs out removing the flag, a garg vampire will be vulnerable to taking internal bleeding when they shouldnt. all limb flags should be managed by traits and the add/remove procs